libs/sdk-ui-charts/src/highcharts/constants/stacking.ts

```diff
@@ -1,3 +1,2 @@
-// (C) 2020 GoodData Corporation
-export const NORMAL_STACK = "normal";
-export const PERCENT_STACK = "percent";
+// (C) 2020-2021 GoodData Corporation
+export type StackingValues = "normal" | "percent";
```

Comment:
Nitpick: the name is a bit misleading (mainly the plural). How about just `Stacking` or `StackingType`?

Reply:
Done: https://github.com/gooddata/gooddata-ui-sdk/pull/1277/commits/b163c6e274ae00d277d59d00935f12dbfdfd60fb